docs/commands.md

```diff
@@ -0,0 +1,13 @@
+# Flutter Commands
+
+## mobile: scheduleAction
```

Comment:
should the name be updated?

Reply:
Yes its a typo. It is 'flutter: activateApp'